src/java.base/share/classes/java/util/Currency.java

```diff
@@ -444,8 +444,9 @@ public static Currency getInstance(Locale locale) {
     public static Set<Currency> getAvailableCurrencies() {
         synchronized(Currency.class) {
             if (available == null) {
+                // reuse same time for all special case cut-over date checks
```

Comment:
The comment is correct, but it might not be necessary, as it would not be significant unless one knows the background

Reply:
Comment removed and year bumped.